reviews three/swagger.yaml

```diff
@@ -0,0 +1,954 @@
+          in: query
+          description: Sets whether or not payment is pending for the draft order.
+          enum:
+          - true
+          - false
+          type: boolean
+        - name: id
+          in: path
+          required: true
+          type: integer
+      responses:
+        '200':
+          description: draft order
+          schema:
+            $ref: '#/definitions/DraftOrder'
+            
+            
+      x-code-samples:
+        - lang: curl
+          source: |
+            curl -X PUT -H "Authorization: Basic YOUR-API-KEY=" "https://your_store_domain.myshopify.com/admin/draft_orders/57760134/complete.json?payment_gateway_id=null&payment_pending=false"    
+            
+            
+  '/admin/draft_orders/#{id}/checkout.json':
+    post:
+      summary: Create or retrieve existing checkout
+      description: >-
+        This call will either retrieve an existing checkout for this draft order or will create a new checkout if none exists previosuly.
+        
+         | `Note`| This API call requires the checkouts_api scope, and is only applicable for users of the Sales Channel SDK.  |
+          |---|-------------|
+        
+        
+      
+      parameters:
+        - name: id
+          in: path
+          required: true
+          type: integer
+      responses:
+        '200':
+          description: Checkout object that was created or retrieved.  Not in scope for Alpha.
+          schema:
+            $ref: '#/definitions/checkout'
+            
+            
+      
+      tags:
+        - Create
+        
+      x-code-samples:
+        - lang: curl
+          source: |
+            curl -X POST -H "Authorization: Basic YOUR-API-KEY=" "https://your_store_domain.myshopify.com/admin/draft_orders/57760134/checkout.json" 
+definitions:
+  DraftOrder:
+    type: object
+    properties:
+      id:
+        type: string
+        description: The `id` of the draft order.
+        readOnly: false
+      order_id:
+        type: string
+        description: The `id` of the order associated to the draft order, once created.
+        readOnly: false
+      name:
+        type: string
+        description: Name of the draft order.
+        readOnly: false
+      customer:
+        type: object
+        $ref: '#/definitions/customer'
+        
+      shipping_address:
+        type: string
+        description: The mailing address to where the draft order will be shipped.
+      billing_address:
+        type: string
+        description: The mailing address associated with the payment method.
+      note:
+        type: string
+        description: The text of an optional note that a shop owner can attach to the draft order.
+      email:
+        type: string
+        description: The email address used for sending notifications.
+      currency:
+        type: string
+        description: The three letter code for the currency to be used for the payment.
+        readOnly: true
```

Comment:
why are you specifying read only only on some fields? The design document has a column for read only.